src/app_service/service/community/service.nim

```diff
@@ -971,11 +971,13 @@ QtObject:
       var pubKeys: seq[string] = @[]
       for pubKey in pubKeysParsed:
         pubKeys.add(pubKey.getStr)
-      let response =  status_go.inviteUsersToCommunity(communityId, pubKeys)
```

Comment:
Shouldn't we remove legacy/unused API from `backend/communities.nim`?

Reply:
Yea definitely! Don't think it has to be done as part of this particular commit